packages/atlas/src/components/_inputs/SubtitlesBox/SubtitlesBox.tsx

```diff
@@ -97,7 +111,7 @@ export const SubtitlesBox: FC<SubtitleBoxProps> = ({
           {languageIso} {type === 'closed-captions' ? '(CC)' : ''}
         </Text>
         <SubtitlesFileName variant="t100" as="p" color="colorText">
-          {!hasFile ? 'Add subtitles file' : file ? file.name : 'subs.vtt'}
+          {!hasFile ? 'Add subtitles file' : file ? file.name : fileName}
```

Comment:
So I asked designers about what should be displayed here for published subtitles: https://joystreamworkspace.slack.com/archives/C039GLGPUQ7/p1661857714024849

They suggested we leave this empty in that case, could you update?

Reply:
@kdembler ok, but should we display the filename once someone uploads subtitles? 
![image](https://user-images.githubusercontent.com/51168865/188395564-c7ce88a4-90d7-4226-80e6-a2740e5672ee.png)
I think we should have the same behavior - don't show filename at all, right now it's confusing to me.

Reply:
Hmm, that's a valid point. Can you tag Kuba on #atlas-design so we can discuss?